web/annotation_editor_layer_builder.css

```diff
@@ -80,18 +81,26 @@
   resize: none;
 }
 
-.annotationEditorLayer .freeTextEditor {
+.annotationEditorLayer :is(.freeTextEditor, .inkEditor, .stampEditor) {
   position: absolute;
   background: transparent;
   border-radius: 3px;
+  z-index: 1;
+  transform-origin: 0 0;
+  cursor: auto;
+}
+
+.annotationEditorLayer :is(.inkEditor, .stampEditor) {
+  overflow: auto;
+}
+
+.annotationEditorLayer .freeTextEditor {
   padding: calc(var(--freetext-padding) * var(--scale-factor));
   resize: none;
   width: auto;
   height: auto;
-  z-index: 1;
   transform-origin: 0 0;
```

Comment:
This line looks unnecessary now, given the rules in https://github.com/mozilla/pdf.js/pull/16585/files#diff-52f5d77845bc000fab73ee0756ed15ce132e88c1893bff5a2268402ea99599aaR84